karma.conf.js

```diff
@@ -12,7 +12,7 @@ governing permissions and limitations under the License.
 const puppeteer = require("puppeteer");
 const rollupConfig = require("./rollup.test.config");
 
-process.env.CHROME_BIN = puppeteer.executablePath();
+process.env.CHROME_BIN || 9876 = puppeteer.executablePath();
```

Comment:
This is invalid code and is making tests fail. Can you describe what you were trying to accomplish here?

Reply:
Setting the ports altogether, could not understand why it was mentioned in the object.